substrate/frame/bounties/src/benchmarking.rs

```diff
@@ -171,27 +171,6 @@ benchmarks_instance_pallet! {
 		ensure!(!T::Currency::free_balance(&beneficiary_account).is_zero(), "Beneficiary didn't get paid");
 	}
 
-	close_bounty_proposed {
```

Comment:
This benchmark case relies on the parameter type `ApproveOrigin` which is deprecated in the `pallet_treasury`. Hence, I simply remove the benchmark case. I wonder if we need a workaround to cover this case or not.